### ISSUE TITLE
Blocked Tirhuta kṣa

### ISSUE THREAD

**Issue by dscorbett** [googlefonts/noto-fonts#1507]
### Font
NotoSansTirhuta-Regular.ttf

### Where the font came from, and when
Site: https://github.com/googlefonts/noto-fonts/blob/fadc9ff5dd82c9ae781d3ba40ae4c238310e06e7/phaseIII_only/unhinted/ttf/NotoSansTirhuta/NotoSansTirhuta-Regular.ttf
Date: 2019-04-23

### Font version
Version 2.001

### Issue
In many Indic scripts, <i>kṣa</i> uses a conjunct that acts like a single indivisible letter. (This is what the 'akhn' feature is for.) In Noto Sans Tirhuta, however, the <i>kṣa</i> can preempted by a preceding conjunct ending in <i>ka</i>. Is that appropriate for Tirhuta?

### Character data
𑒓𑓂𑒏𑓂𑒭
U+11493 TIRHUTA LETTER NGA
U+114C2 TIRHUTA SIGN VIRAMA
U+1148F TIRHUTA LETTER KA
U+114C2 TIRHUTA SIGN VIRAMA
U+114AD TIRHUTA LETTER SSA
𑒭𑓂𑒏𑓂𑒭
U+114AD TIRHUTA LETTER SSA
U+114C2 TIRHUTA SIGN VIRAMA
U+1148F TIRHUTA LETTER KA
U+114C2 TIRHUTA SIGN VIRAMA
U+114AD TIRHUTA LETTER SSA

### Screenshot
![𑒓𑓂𑒏𑓂𑒭 𑒭𑓂𑒏𑓂𑒭](https://user-images.githubusercontent.com/1124347/57538734-0a8d8680-7317-11e9-8d83-a731f16a4dbb.png)

**Comment by punchcutter** [comment]
Yes, that isn't right. There could be ṅkṣa or ṣkṣa conjuncts here if those exist in Tirhuta (I don't see them yet in resources I have), but otherwise kṣa should take precedence over any other conjuncts.
